### PR TITLE
add link to state-scripts example directory

### PR DESCRIPTION
**Pull request by mirzak**
We store a couple of state-script examples (code) in
the Mender client repository.

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>

### CODE REVIEW

File: 04.Artifacts/07.State-scripts/docs.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ State scripts are allowed to return a specific error code, in order to rerun at
 
 Mender users will probably come up with a lot of interesting use cases for state scripts, and we will cover some well-known ones below for inspiration.
 
+<!--AUTOVERSION: "mender/tree/%"/ignore-->
+You can find code examples in the [Mender client source repository](https://github.com/mendersoftware/mender/tree/master/examples/state-scripts) and if you have implemented an interesting use-case we encourage you to share it with the community at the linked location.
```
Comment (lluiscampos) [inline 04.Artifacts/07.State-scripts/docs.md]: What does it mean "at the linked location". It is quite unclear to me

Comment (estenberg) [reply 04.Artifacts/07.State-scripts/docs.md]: maybe reword to "encourage you to create a PR so all of the community can benefit".

Comment (mirzak) [reply 04.Artifacts/07.State-scripts/docs.md]: Updated
